tools/clang/test/taef_exec/lit.cfg

```diff
@@ -0,0 +1,39 @@
+# -*- Python -*-
+
+# Configuration file for the 'lit' test runner.
+
+import os
+import platform
+
+import lit.formats
+import lit.util
+
+# name: The name of this test suite.
+config.name = 'clang-taef-exec'
+
+# Check that the object root is known.
+te = getattr(config, 'te', None)
+if te is None:
+    # Otherwise, we haven't loaded the site specific configuration (the user is
+    # probably trying to run on a test file directly, and either the site
+    # configuration hasn't been created by the build system, or we are in an
+    # out-of-tree build situation).
+
+    # Check for 'clang_unit_site_config' user parameter, and use that if available.
+    site_cfg = lit_config.params.get('clang_taef_exec_site_config', None)
+    if site_cfg and os.path.exists(site_cfg):
+        lit_config.load_config(config, site_cfg)
+        raise SystemExit
+
+# TAEF only runs on Windows
+if platform.system() != 'Windows':
+  config.unsupported = True
+else:
+  test_dll = os.path.join(config.llvm_obj_root, config.llvm_build_mode, 'bin', 'ExecHLSLTests.dll')
+
+  hlsl_data_dir = os.path.join(config.llvm_src_root, 'tools', 'clang', 'test', 'HLSL')
```

Comment:
Is `hlsl_data_dir` unused?  Shouldn't it contribute to `extra_params`?

Reply:
with hlsl_data_dir, a lot of execution test fails.

Reply:
Without it, does it silently skip or pass without actually running?  I don't think we want this in this state.

Reply:
Found the issue, used wrong hlsl_data_dir :(